app/models/energy/schedulefixedinterval.py

```diff
@@ -0,0 +1,44 @@
+"""Schedule Fixed Interval Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+import datetime
+from app.models.common.datetime import Date
+
+
+class ScheduleFixedInterval(BaseModel):
+    """Used to specify a start date and a list of values for a period of analysis."""
+
+    type: Enum('schedulefixedinterval', {'type': 'schedulefixedinterval'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    start_date: Date
```

Comment:
@chriswmackey I believe Ruby has a datetime module. Should I keep leave the start_date the way it is? 
The @validator in `datetime.py` ensures the start date is not 02/29.

Reply:
This allows each schedule in the model to have a different start date.  Difficult situations could arise like:

- Schedule A starts on 1/1 and Schedule B starts on 6/1
- Schedule A starts on 1/1 of a leap year and Schedule B starts on 1/1 of a non leap year
- Schedule A starts at 6/1 but simulation starts on 1/1

Maybe there could be model level validation that all schedules have the same start date?  Or some way to adjust all schedules to the simulation period on translation (filling with 0's as needed)?

Reply:
Should this object have an interpolate_to_timestep field?

Reply:
I'm not sure about adding the `interpolate_to_timestep`  field because all values entered are for a 24 hour period.

Reply:
Interpolate to timestep is really about if values are interpolated between the until times or not.  Here is the relevant field in E+ https://bigladdersoftware.com/epx/docs/8-2/input-output-reference/group-schedules.html#field-interpolate-to-timestep

Reply:
@macumber ,
We will have some validation methods on the Python end for this so I would not get too concerned about it here in the schema.  However, this does affect how you will implement  `ScheduleFixedInterival` to use CSV in OpenStudio SDK since (as I recall) the E+ `ScheduleFile` only accepts a full 8760 values in the CSV.  So you will probably have to fill some spaces in the CSV with 0's to get it to work.